extensions/amp-web-push/0.1/amp-web-push-widget.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import {Layout} from '../../../src/layout';
+
+/**
+ * @fileoverview
+ * A <web-push-widget> that shows or hides based on the page user's subscription
```

Comment:
Ditto here: I think calling them `amp-web-push` and `amp-web-push-widget` would clear up all these things.